### DISCUSSION TITLE
help:cc

### DISCUSSION THREAD

**Discussion by fcoagz** [eternnoir/pyTelegramBotAPI#1943] [Q&A]
Hello! I would like to share my bot in your repository, I have the source code and everything, but I really don't know if I should go to a link for you to accept or how is it? https://github.com/fcoagz/DownloadVideoBOT

**Comment by coder2020official** [comment]
Make a PR

**Reply by fcoagz** [reply]
what is PR?

**Reply by dimanjy** [reply]
Its Pull Request

**Reply by fcoagz** [reply]
tankiu<3333
